src/SchemaDefinition/CoreDefinition.php

```diff
@@ -0,0 +1,168 @@
+<?php declare(strict_types = 1);
+
+namespace Apitte\OpenApi\SchemaDefinition;
+
+use Apitte\Core\Schema\Endpoint;
+use Apitte\Core\Schema\EndpointParameter;
+use Apitte\Core\Schema\EndpointRequest;
+use Apitte\Core\Schema\EndpointResponse;
+use Apitte\Core\Schema\Schema as ApiSchema;
+
+class CoreDefinition implements IDefinition
+{
+
+	/** @var ApiSchema */
+	protected $schema;
+
+	public function __construct(ApiSchema $schema)
+	{
+		$this->schema = $schema;
+	}
+
+	/**
+	 * @return mixed[]
+	 */
+	public function load(): array
+	{
+		return ['paths' => $this->createPaths()];
+	}
+
+	/**
+	 * @return mixed[]
+	 */
+	protected function createPaths(): array
+	{
+		$paths = [];
+		foreach ($this->getEndpoints() as $endpoint) {
+			$operations = [];
+			foreach ($endpoint->getMethods() as $method) {
+				$operations[strtolower($method)] = $this->createOperation($endpoint);
+			}
+			$paths[$endpoint->getMask() ?? ''] = $operations;
```

Comment:
Should work with null too http://sandbox.onlinephpfunctions.com/code/6aa7b4a710cf2ee3b8820357ebeb58b27a2981b0